images/techdocs/context/mkdocs.yml

```diff
@@ -1,4 +1,7 @@
 site_name: Baseline
+repo_name: !ENV [REPO_NAME, null]
+repo_url: !ENV [REPO_URL, null]
+edit_url: !ENV [EDIT_URL, null]
```

Comment:
I haven't seen this syntax before. Is this specific to mkdocs?

Reply:
I think so

Reply:
My yaml isn't perfect, but it looks to me like `!ENV` is a custom type `ENV`.